pyproject.toml

```diff
@@ -11,7 +11,7 @@ pymongo = "3.12.1"
 python = "^3.9"
 requests = "2.26.0"
 django-split-settings = "^1.1.0"
-Django = "^3.2.9"
+Django = "3.2.11"
```

Comment:
@olegtropinin next time keep it `Django = "^3.2.11"`. Please, read about `^` meaning for poetry if you have not. The same for `pytest-django`